Client/src/components/common/Course/index.css

```diff
@@ -0,0 +1,20 @@
+.coursCard {
+  border-radius: 2px;
+  box-shadow: 0px 0px 1px rgba(0, 0, 0, 0.25);
+  width: 80vw;
+  height: 20vh;
+  margin: 30px auto;
+  padding: 10px;
+}
+
+.coursCard-titel {
+  text-align: left;
+  color: #06468F;
+}
+
+.coursCard-desc {
+  text-align: left;
```

Comment:
instead of repeat this line you can  it in the parent of them

Reply:
I delete it